### PR TITLE
Added AppCenterSDK file for mac (removed distribution features)

### PR DESCRIPTION
**Pull request by UrsKahmann**

### CODE REVIEW

File: AppCenter/AppCenterSDK+Mac.swift [hunk 1]
```diff
@@ -0,0 +1,164 @@
+//
```
Comment (Tomusm) [inline AppCenter/AppCenterSDK+Mac.swift]: Do we need that file ? What are the differences with `AppCenterSDK+HiDrive` ?
I would assume that we use the same implementation, better reduce the number of files to maintain.

Comment (UrsKahmann) [reply AppCenter/AppCenterSDK+Mac.swift]: The problem is, that the AppCenterDistribute module is not available for mac .. and as you don't need any distribution logic i just removed the import statement and any associated code

Comment (Tomusm) [reply AppCenter/AppCenterSDK+Mac.swift]: What about extracting the Distribution code in an extension in the same file, and surrounding it by `#if !os(macOS)` ?

(Not sure if it actually works, just a random idea)

Comment (UrsKahmann) [reply AppCenter/AppCenterSDK+Mac.swift]: doesn'T sound that bad i can try that

Comment (UrsKahmann) [reply AppCenter/AppCenterSDK+Mac.swift]: done!

**Pull request merged**